docs/scheduling.md

```diff
@@ -16,8 +16,7 @@ In our experience, manual management of concurrency via obtaining and releasing
 
 [AbstractBackgroundJob](../src/infrastructure/AbstractBackgroundJob.ts) is the base class that you should extend when implementing your background jobs.
 
-You should register all of your jobs to `app.scheduler` in [jobs.ts](../src/modules/jobs.ts) to ensure that they are launched when application starts.
-If you choose to do your own job registration instead of using `jobs.ts`, note that it needs to happen within the `app.after(() => {})` block.
+You should use `eagerInject: 'register` in DI configuration all of your jobs in [diConfig.ts](../src/infrastructure/diConfig.ts) to ensure that they are launched when application starts.
```

Comment:
`You should use `eagerInject: 'register` in DI configuration all of your jobs`
Did you miss a word there?

Reply:
fixed, thanks